src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -197,6 +196,13 @@ private static List<Species> CopyInitialPopulationsToResults(SimulationConfigura
                 niches.Add(new HeterotrophicFoodSource(patch, currentSpecies, cache));
             }
 
+            var creatureNiche = new Dictionary<MicrobeSpecies, Dictionary<MicrobeSpecies, float>>();
```

Comment:
I'm confused what this is capturing.

Reply:
It's capturing how much energy each species gets from the niche for each other creature. The more often you eat a creature, the larger your effect on the mortality rate. Improved name suggestions are welcome :) I should also add a comment.